src/macrofier/types.mjs

```diff
@@ -151,21 +186,7 @@ const insertObjectMacros = (content, schema, module, title, property, options) =
     const template = getTemplate(path.join(options.templateDir, 'property' + (templateType ? `-${templateType}` : ''))).replace(/\n/gms, indent + '\n')
   
     const properties = []
-  
-    if (schema.additionalProperties && (typeof schema.additionalProperties === 'object')) {
-      const template = getTemplate(path.join(options.templateDir, 'additionalProperties'))
-      let type = getSchemaType(schema.additionalProperties, module, options2)
-      const shape = getSchemaShape(schema.additionalProperties, module, options2)
-      properties.push(template
-        .replace(/\$\{property\}/g, '')
-        .replace(/\$\{Property\}/g, '')
-        .replace(/\$\{shape\}/g, shape)
-        .replace(/\$\{parent\.title\}/g, title)
-        .replace(/\$\{title\}/g, type)
-        .replace(/\$\{delimiter\}(.*?)\$\{end.delimiter\}/g, '')
-        .replace(/\$\{if\.optional\}(.*?)\$\{end\.if\.optional\}/g, ''))
-    }
-  
+
```

Comment:
Shouldn't we be calling  `insertObjectAdditionalProperitesMacros` here, with the same `if (schema.additionalProperties...` block?

Reply:
I updated code now to call insertObjectAdditionalProperitesMacros now